### PR TITLE
user guide update & change version to 7.1.xx

### PR DESCRIPTION
**Pull request by inponomarev**
## Overview

Edited User Guide in order to remove outdated information and to catch up with recent changes.

---

### Checklist

- [ ] Change is covered by automated tests.
- [ ] JavaDoc is provided for all new/changed public APIs.
- [x] User Guide is updated.
- [ ] CI builds pass.

### CODE REVIEW

File: celesta-documentation/src/main/asciidoc/0000_Celesta.adoc [hunk 1]
```diff
@@ -15,23 +15,25 @@ include::1010_what_is_celesta.adoc[leveloffset=+2, tag="intro"]
 
 == Часть 1. Быстрый старт
 
-=== Maven-зависимости
+=== Демонстрационный пример
 
 include::1012_maven_dependencies.adoc[leveloffset=+2, tag="maven_dependencies"]
```
Comment (packpaul) [inline celesta-documentation/src/main/asciidoc/0000_Celesta.adoc]: Why not `1012_demonstration.adoc`?

File: celesta-documentation/src/main/asciidoc/1013_maven_plugin.adoc [hunk 2]
```diff
@@ -13,24 +13,58 @@ include::_doc_general_attributes.adoc[]
...
-** На фазе `generate-test-resources` -- из `src/test/celestasql` в `generated-test-resources/score`
+** На фазе `generate-test-resources` -- из `testScores` в `generated-test-resources/score`
 
 3. Генерирует индекс `score.files` с перечнем относительных путей к .sql-файлам в ресурсах:
 
 ** На фазе `generate-resources` -> `generated-resources/score/score.files`
 ** На фазе `generate-test-resources` ->`generated-test-resources/score/score.files`
 
-Данные файлы используются для score discovery во время выполнения Celesta, избавляя от необходимости в runtime исследовать содержимое всех jar-файлов на classpath.
+Эти файлы используются для во время выполнения Celesta для того, чтобы найти пути ко всем CelestaSQL-файлам внутри jar-файлов на classpath.
```
Comment (packpaul) [inline celesta-documentation/src/main/asciidoc/1013_maven_plugin.adoc]: "используются для во" -> "используются во"

**Review by packpaul — CHANGES_REQUESTED** [review]

File: celesta-documentation/src/main/asciidoc/0000_Celesta.adoc (hunk 1 above)
Comment (inponomarev) [reply celesta-documentation/src/main/asciidoc/0000_Celesta.adoc]: fixed

File: celesta-documentation/src/main/asciidoc/1013_maven_plugin.adoc (hunk 2 above)
Comment (inponomarev) [reply celesta-documentation/src/main/asciidoc/1013_maven_plugin.adoc]: fixed

**Review by packpaul — APPROVED** [review]

**Pull request merged**